refget/agents.py

```diff
@@ -573,6 +574,20 @@ def compare_digests(self, digestA, digestB):
         B = self.seqcol.get(digestB, return_format="level2")
         return compare_seqcols(A, B)
 
+    def calc_similarities(self, digestA, digestB):
+        A = self.seqcol.get(digestA, return_format="level2")
+        B = self.seqcol.get(digestB, return_format="level2")
+        return calc_jaccard_similarities(A, B)
```

Comment:
I know that other functions don't have docstring, but it would be nice to add 1) type of data user should provide. 2) docstring